src/lib.rs

```diff
@@ -12,15 +12,27 @@ pub enum Error {
     SpawnChild(std::io::Error),
 }
 
-pub fn relaunch_if_pid1() -> Result<(), Error> {
-    if std::process::id() == 1 {
+pub fn relaunch_if_pid1(option: Pid1Settings) -> Result<(), Error> {
+    let pid = std::process::id();
+    if pid == 1 {
         let child = relaunch()?;
+        if option.log {
+            println!("pid1-rs: Process running as PID 1");
```

Comment:
```suggestion
            eprintln!("pid1-rs: Process running as PID 1");
```

Reply:
Now I see that you used eprintln below, any reason not to use it in pid1? I'm thinking that the normal case for this will be everything running in a single Docker container, and we'd want to separate out the log output from real application output.

Reply:
Sounds good, I have fixed it. Thanks!